packages/libraries/client/README.md

```diff
@@ -48,6 +48,35 @@ const server = createYoga({
 server.start()
 ```
 
+### GraphQL Yoga and Cloudflare Workers
+
+If you're using Cloudflare Workers, you can use the following code:
+
+```typescript
+import { useYogaHive, createYogaHive, useYogaHive } from '@graphql-hive/client'
+import { createYoga } from 'graphql-yoga'
+
+const hive = createYogaHive({
+  enabled: true, // Enable/Disable Hive Client
+  token: 'YOUR-TOKEN',
```

Comment:
For cloudflare workers this needs to happen within the fetch handler as the env variables are not available globally. Could you consider an alternative API to pass the token down ? Maybe try to read it automatically from the cloudflare env assuming a specific variable name ?

Reply:
You can run `createYogaHive` within fetch, it has no overhead really